experiments/coupled_simulation/ocean_sea_ice_coupled_simulation.jl

```diff
@@ -18,15 +18,14 @@ arch = CPU()
 
 depth = 1000meters
 Nz    = 10
-h     = 3
 
-r_faces = ClimaOcean.exponential_z_faces(; Nz, h, depth)
-z_faces = MutableVerticalDiscretization(r_faces)
+z = ExponentialInterfaces(Nz, depth; scale=0.3*depth)
+z = MutableVerticalDiscretization([z(k) for k in 1:Nz+1])
```

Comment:
Let's avoid this pattern and just use `MutableVerticalDiscretization(z)` as we do normally

Reply:
I wasn't sure it would work; I'll try it out